packages/in_app_purchase/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.1+1
+
+* Update readme
```

Comment:
Please describe the actual change to the README

Reply:
Thanks!

Reply:
You've just changed the capitalization; please actually describe what in the README you fixed rather than just saying that the README was updated. Just as we describe code fixes, rather than saying "updated the code".